src/collect.js

```diff
@@ -5,6 +5,7 @@
  * https://github.com/facebook/nuclide/blob/master/pkg/nuclide-flow-rpc/lib/FlowRoot.js
  * https://github.com/ptmt/tryflow/blob/gh-pages/js/worker.js
  */
+import pathModule from 'path';
```

Comment:
I did it at first, but eslint complained: `path is already declared in the upper scope.  no-shadow`